README.md

```diff
@@ -59,7 +59,7 @@ This is To do list Web application. With this Web Application you can add your d
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-To see my portfolio click [here]()
+To see my portfolio click [here]( https://esmatullah12.github.io/to-do-list/)
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly fix the deployment or remove the section completely

Reply:
- Hey, while you've made changes to the edit functionality, please keep in mind that you've not made the changes as pointed out by the previous reviewer concerning the deployment in the readme. since your app works now, I'd overlook it since this is your last submission, but please remove the section if you won't add a working demo link. Please fix this issue before you merge. it won't be overlooked in your next project milestone. :pray: